tests/test_metric_output.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import yaml
+
+
+class TestMetricOutput:
+    def test_vcsa_data_comparison(self, setup_vcsa_url):
+        response = requests.get(setup_vcsa_url)
+        metrics = self.process_response_data(response)
+
+        with open('tests/vcsa_output_comparison.yaml', 'r') as fp:
```

Comment:
Would put into data dir where you have put the other 'static' files already

Reply:
Will add the file to mockingServer/data

Reply:
done